src/app/sidenav/sidenav.component.html

```diff
@@ -1,3 +1,10 @@
-<div class="sidenav--content__container">
+<div class="sidenav--content__container" fxFlex fxLayout="column">
+
```

Comment:
If we can, no white space rows between intended html tags - only white space separation for same tags inline if that makes sense